BEIMA.Client/src/App.js

```diff
@@ -25,6 +26,7 @@ const App = () => {
       <Routes>
         <Route path="/" element={<AppLayout />}>
           <Route index element={<HomePage />} />
+          <Route path="login" element={<LoginPage/>}/>
```

Comment:
We should make the login page the landing page

Reply:
I that will likely depend on their login session or whatnot, so I would suggest waiting to do that until we actually have logins working.